src/classes/PlayerBase.js

```diff
@@ -0,0 +1,33 @@
+import { InteractableGameObject } from "./InteractableGameObject.js";
+import { Layer } from "./managers/Layer.js";
+import { NewMediaManager } from "./managers/NewMediaManager.js";
+
+export class PlayerBase extends InteractableGameObject {
+    constructor(xLocation, yLocation) {
+        /**
+         * The width, height, angle (which doesn't really apply), sprite, velocityX, velocityY, collisionRadius, and mass are always the same for a PlayerBase.
+         * While the xLocation and yLocation values are also always the same when a new PlayerBase is created, leaving that as a value to pass in for now.
```

Comment:
Technically not true since player base location depends on canvas size

- [ ] Update comment about location of player base